docs/docs/contributions/development/setting-up-pants.mdx

```diff
@@ -61,32 +61,30 @@ To free up space, run `rm -rf src/rust/engine/target`.
 Warning: this will cause Rust to recompile everything.
 :::
 
-## Step 3: (Optional) Set up a Git hook
+## Step 3: Set up a pre-push Git Hook
```

Comment:
Removing "Optional"

Seems weird to write that in the development docs. For the people who develop often, they're used to the workflow and can do as they please anyways.

For those who are new to committing to Pants, this feels like a simple thing that would be nice to save some CI loops.